fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -56,6 +57,13 @@ import com.microsoft.fluentui.theme.token.controlTokens.AppBarTokens
  * @param accessoryDelta Ratio of opening of accessory View. Used for Shychrome and other animations. Default: [1.0F]
  * @param appBarTokens Optional Tokens for App Bar to customize it. Default: [null]
  */
+
+// TAGS FOR TESTING
+const val APP_BAR = "App bar"
+const val APP_BAR_SUBTITLE = "Subtitle"
```

Comment:
add "App Bar" as prefix in value to avoid name classes while finding items in UI.